engines/wintermute/base/gfx/opengl/base_render_opengl3d.cpp

```diff
@@ -74,6 +74,21 @@ void BaseRenderOpenGL3D::setSpriteBlendMode(Graphics::TSpriteBlendMode blendMode
 	}
 }
 
+bool BaseRenderOpenGL3D::enableShadows() {
+	warning("BaseRenderOpenGL3D::enableShadows not implemented yet");
+	return true;
+}
+
+bool BaseRenderOpenGL3D::disableShadows() {
+	warning("BaseRenderOpenGL3D::disableDhadows not implemented yet");
+	return true;
+}
+
+bool BaseRenderOpenGL3D::stencilSupported() {
+	// assume that we have a stencil buffer
```

Comment:
We could perhaps check this at the initialization point, as various context details are easily checked then. Would be OK to error out on missing stencil-buffer.

Reply:
Discussed this on Discord already, although a precise solution is still open. Would prefer setting a flag at context initialization.

Reply:
This is fine to handle in the next PR.